ui/packages/catalog-ui-search/src/main/webapp/component/result-item/result-item.view.js

```diff
@@ -185,6 +185,9 @@ const ResultItemView = Marionette.LayoutView.extend({
                 title="Follow external links"
                 data-help="Follow external links."
               />
+              <div className="result-validation">
```

Comment:
Why do we need the className here?

Reply:
My extension only returns a button component, so there is some styling needed here for it to look correct. Should this div and styling be added in the extension instead?